vxquery-benchmark/src/main/resources/noaa-ghcn-daily/scripts/weather_convert.py

```diff
@@ -0,0 +1,830 @@
+#!/usr/bin/env python
```

Comment:
It is a new file that contains an abstract converter class. Then, according to which converter we need the methods are implemented in the corresponding converter. The two converters classes are, also contained in this file.